documentation/modules/overview/con-configuration-points-mirrormaker.adoc

```diff
@@ -2,42 +2,8 @@
 //
 // overview/assembly-configuration-points.adoc
 
-[id="con-configuration-points-mm2-{context}"]
```

Comment:
You just changed this here from Mirror Maker 2 to Mirror MAker 1. Was that intentional?